cmd/openshift-compatibility-gen/generate/comment/generator.go

```diff
@@ -201,8 +201,19 @@ func (g *compatibilityLevelCommentGenerator) applyCompatibilityLevelComment() ds
 			g.err = fmt.Errorf("%s: APIs whose versions do not conform to kube apiVersion format cannot be exposed: the %s API must be tagged with +%s", apiTypeName, apiTypeName, internalTagName)
 			return false
 		case ga && level != 1:
-			g.err = fmt.Errorf("%s: generally available APIs must be supported for a minimum of 12 months", apiTypeName)
-			return false
+			switch {
+			case apiTypeName == "PodSecurityPolicySubjectReview" && level == 2:
+				// Documented special case, this api is level 2
+			case apiTypeName == "PodSecurityPolicySelfSubjectReview" && level == 2:
+				// Documented special case, this api is level 2
+			case apiTypeName == "PodSecurityPolicyReview" && level == 2:
+				// Documented special case, this api is level 2
+			case apiTypeName == "RangeAllocation" && level == 4:
```

Comment:
i'd love to make these checks more precise (by including api group in the match) but that information doesn't seem to be easily extractable.  If someone knows of an easy way, i'm happy to update it.